src/styles/feature-item.scss

```diff
@@ -0,0 +1,9 @@
+.feature-link {
+  margin-left: -40px;
+  list-style: none;
+  margin-bottom: 30px;
```

Comment:
Please use Amplify UI space tokens here instead of one-off pixel values so that we're keeping consistent spacing throughout the UI. For example:
```
margin-bottom: var(--amplify-space-large);
```
The same with anywhere else we're using margins or other spacing properties.